sbi/inference/potentials/score_fn_iid.py

```diff
@@ -106,6 +106,20 @@ def __init__(
         self.vector_field_estimator = vector_field_estimator.to(device).eval()
         self.prior = prior
 
+    def to(self, device: Union[str, torch.device]) -> None:
+        """
+        It moves score_estimator and prior to the given device.
```

Comment:
Sorry for being picky here, but ideally the docstring use the same form throughout, e.g., 
```suggestion
        Moves score_estimator and prior to the given device.
```